flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDialectITCase.java

```diff
@@ -794,6 +795,33 @@ public void testShowPartitions() throws Exception {
         assertThat(partitions.toString()).contains("dt=2020-04-30 01:02:03/country=china");
     }
 
+    @Test
+    public void testSetCommand() throws Exception {
```

Comment:
Is the user able to `SET key=value -v`? Also, add a case to check whether SET works.